app/d3plus.js

```diff
@@ -150,6 +150,9 @@ export default {
       }
     },
     color: colors.colorScaleGood,
+    colorMin: "#b0cde1",
+    colorMid: "#4c96cb",
+    colorMax: "#004374",
```

Comment:
please use the imported `colors` Object for these references, rather than hardcoding hex values

Reply:
done